pkg/interactive-installer/README.md

```diff
@@ -0,0 +1,31 @@
+# EVE-OS interactive installer
+
+## Description
+
+The EVE-OS interactive installer provides a user-friendly text-based user interface (TUI) to configure the parameters of the installation. The primary goal of this installer mode is to facilitate the installation of EVE-OS on-edge devices while providing flexibility, and automation.
+The EVE-OS interactive installer provides a user-friendly environment for installing EVE-OS. Utilizing a TUI, allows users to configure installation parameters with ease. The primary goal of this installer is to simplify the installation process on edge devices while offering flexibility and automation, similar to the installation wizards used for setting up new Wi-Fi routers.
```

Comment:
I commented earlier - maybe comment was lost?
You have two paragraphs which are almost identical - please make it into one paragraph.